addons/pos_event/models/event_event.py

```diff
@@ -0,0 +1,16 @@
+from odoo import api, fields, models
+
+
+class Event(models.Model):
+    _name = 'event.event'
+    _inherit = ['event.event', 'pos.load.mixin']
+
+    image_1024 = fields.Image("Image", max_width=1024, max_height=1024)
```

Comment:
The specs ask you to use the header image

Reply:
Ok, would still be nice to change string and add an help on the field to explicitly say it will be used in pos.
Could be improved post freeze if no time.